From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Tue, 13 Oct 2026 21:20:17 +0000
Subject: [PATCH] Revert "[bugfix] Fix chrome target generation when secondary
 ABI is disabled"

This reverts commit 099337df47a84e7544810997e4dd0d5fb3caf7e3.
---
 chrome/android/BUILD.gn | 56 ++++++++++++++++++-----------------------
 1 file changed, 24 insertions(+), 32 deletions(-)

diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
index 46782baba7fb4..9a30c0b8902e8 100644
--- a/chrome/android/BUILD.gn
+++ b/chrome/android/BUILD.gn
@@ -3589,50 +3589,42 @@ if (_is_default_toolchain &&
     deps = []
     if (enable_libs_and_assets_verification) {
       deps += [
+        ":trichrome_chrome_32_64_bundle_validate_libs_and_assets",
+        ":trichrome_chrome_32_bundle_validate_libs_and_assets",
+        ":trichrome_chrome_64_32_bundle_validate_libs_and_assets",
         ":trichrome_chrome_64_bundle_validate_libs_and_assets",
+        ":trichrome_library_32_64_apk_validate_libs_and_assets",
+        ":trichrome_library_32_apk_validate_libs_and_assets",
+        ":trichrome_library_64_32_apk_validate_libs_and_assets",
         ":trichrome_library_64_apk_validate_libs_and_assets",
+        "//android_webview:trichrome_webview_32_64_bundle_validate_libs_and_assets",
+        "//android_webview:trichrome_webview_32_bundle_validate_libs_and_assets",
+        "//android_webview:trichrome_webview_64_32_bundle_validate_libs_and_assets",
         "//android_webview:trichrome_webview_64_bundle_validate_libs_and_assets",
       ]
-      if (defined(android_app_secondary_abi)) {
-        deps += [
-          ":trichrome_chrome_32_64_bundle_validate_libs_and_assets",
-          ":trichrome_chrome_32_bundle_validate_libs_and_assets",
-          ":trichrome_chrome_64_32_bundle_validate_libs_and_assets",
-          ":trichrome_library_32_64_apk_validate_libs_and_assets",
-          ":trichrome_library_32_apk_validate_libs_and_assets",
-          ":trichrome_library_64_32_apk_validate_libs_and_assets",
-          "//android_webview:trichrome_webview_32_64_bundle_validate_libs_and_assets",
-          "//android_webview:trichrome_webview_32_bundle_validate_libs_and_assets",
-          "//android_webview:trichrome_webview_64_32_bundle_validate_libs_and_assets",
-        ]
-      }
     }
     if (enable_manifest_verification) {
       deps += [
+        ":trichrome_chrome_32_64_bundle__base_bundle_module_validate_android_manifest",
+        ":trichrome_chrome_32_bundle__base_bundle_module_validate_android_manifest",
+        ":trichrome_chrome_64_32_bundle__base_bundle_module_validate_android_manifest",
+        ":trichrome_chrome_64_32_bundle_validate_manifests",
+        ":trichrome_chrome_64_32_bundle_validate_proguard_config",
         ":trichrome_chrome_64_bundle__base_bundle_module_validate_android_manifest",
+        ":trichrome_library_32_64_apk_validate_android_manifest",
+        ":trichrome_library_32_apk_validate_android_manifest",
+        ":trichrome_library_64_32_apk_validate_android_manifest",
         ":trichrome_library_64_apk_validate_android_manifest",
+        "//android_webview:system_webview_32_64_base_bundle_module_validate_android_manifest",
+        "//android_webview:system_webview_32_base_bundle_module_validate_android_manifest",
+        "//android_webview:system_webview_64_32_base_bundle_module_validate_android_manifest",
         "//android_webview:system_webview_64_base_bundle_module_validate_android_manifest",
+        "//android_webview:trichrome_webview_32_64_base_bundle_module_validate_android_manifest",
+        "//android_webview:trichrome_webview_32_base_bundle_module_validate_android_manifest",
+        "//android_webview:trichrome_webview_64_32_base_bundle_module_validate_android_manifest",
+        "//android_webview:trichrome_webview_64_32_bundle_validate_proguard_config",
         "//android_webview:trichrome_webview_64_base_bundle_module_validate_android_manifest",
       ]
-      if (defined(android_app_secondary_abi)) {
-        deps += [
-          ":trichrome_chrome_32_64_bundle__base_bundle_module_validate_android_manifest",
-          ":trichrome_chrome_32_bundle__base_bundle_module_validate_android_manifest",
-          ":trichrome_chrome_64_32_bundle__base_bundle_module_validate_android_manifest",
-          ":trichrome_chrome_64_32_bundle_validate_manifests",
-          ":trichrome_chrome_64_32_bundle_validate_proguard_config",
-          ":trichrome_library_32_64_apk_validate_android_manifest",
-          ":trichrome_library_32_apk_validate_android_manifest",
-          ":trichrome_library_64_32_apk_validate_android_manifest",
-          "//android_webview:system_webview_32_64_base_bundle_module_validate_android_manifest",
-          "//android_webview:system_webview_32_base_bundle_module_validate_android_manifest",
-          "//android_webview:system_webview_64_32_base_bundle_module_validate_android_manifest",
-          "//android_webview:trichrome_webview_32_64_base_bundle_module_validate_android_manifest",
-          "//android_webview:trichrome_webview_32_base_bundle_module_validate_android_manifest",
-          "//android_webview:trichrome_webview_64_32_base_bundle_module_validate_android_manifest",
-          "//android_webview:trichrome_webview_64_32_bundle_validate_proguard_config",
-        ]
-      }
     }
   }
 }
